content/id/docs/concepts/services-networking/dual-stack.md

```diff
@@ -0,0 +1,144 @@
+---
+title: _Dual-stack_ IPv4/IPv6 
+feature:
+  title: _Dual-stack_ IPv4/IPv6 
+  description: >
+    Pengalokasian alamat IPv4 dan IPv6 untuk Pod-Pod dan layanan-layanan
+
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+_Dual-stack_ IPv4/IPv6 memungkinkan pengalokasian alamat IPv4 dan IPv6 untuk
+{{< glossary_tooltip text="Pod" term_id="pod" >}} dan {{< glossary_tooltip text="layanan" term_id="service" >}}.
+
+Jika kamu mengaktifkan jaringan _dual-stack_ IPv4/IPv6 untuk klaster Kubernetes 
+kamu, klaster akan mendukung pengalokasian kedua alamat IPv4 dan IPv6 secara 
+bersamaan.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Fitur-fiture yang didukung
+
+Mengaktifkan _dual-stack_ IPv4 / IPv6 pada klaster Kubernetes kamu untuk 
+menyediakan fitur-fitur berikut ini:
+
+   * Jaringan Pod _dual-stack_ (pengalokasian sebuah alamat IPv4 dan IPv6 
+     untuk setiap Pod)
+   * Layanan yang mendukung IPv4 dan IPv6 (setiap Layanan hanya untuk satu 
+     keluarga alamat)
+   * Pod routing untuk keluar klaster (misalnya Internet) melalui antarmuka 
+     IPv4 dan IPv6
+
+## Prasyarat
+
+Prasyarat berikut diperlukan untuk menggunakan _dual-stack_ IPv4/IPv6 pada 
+klaster Kubernetes :
+
+   * Kubernetes versi 1.16 atau yang lebih baru
+   * Dukungan dari penyedia layanan akan jaringan _dual-stack_ (Penyedia layanan
+     _cloud_ atau yang lainnya harus dapat memberikan Node Kubernetes dengan 
+     antarmuka jaringan IPv4/IPv6 yang dapat dirutekan)
+   * Sebuah _plugin_ jaringan yang mendukung _dual-stack_ (seperti Kubenet atau Calico)
+   * Kube-proxy berjalan dalam mode IPVS
+
+## Mengaktifkan _dual-stack_ IPv4/IPv6
+
+Untuk mengaktifkan _dual-stack_ IPv4/IPv6, aktifkan `IPv6DualStack` [gerbang fitur](../en/docs/reference/command-line-tools-reference/feature-gates/) 
```

Comment:
This is something that always confused me because many of the links are not yet available on the Indonesia page.
So, it will leave it as a "broken link" or we need to refer to the English page.

What is your thought @girikuncoro ?

Reply:
Aside: there is upcoming work to make this kind of link pick the localized page if it exists or the English original as a fallback.

Reply:
> Aside: there is upcoming work to make this kind of link pick the localized page if it exists or the English original as a fallback.

It will be a great solution!